_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,279 @@
+
+
+// ----------------------------
+// Polyfill (Flex)
+// ----------------------------
+%justify-right-flex {
+    -webkit-box-pack: flex-end;
+    -webkit-justify-content: flex-end;
+    -moz-box-pack: flex-end;
+    -ms-flex-pack: flex-end;
+    justify-content: flex-end;
+}
+
+%justify-left-flex {
+    -webkit-justify-content: flex-start;
+    -webkit-box-pack: flex-start;
+    -moz-box-pack: flex-start;
+    -ms-flex-pack: flex-start;
+    justify-content: flex-start;
+}
+
+%align-center-flex {
+    -webkit-align-items: center;
+    -webkit-flex-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
+
+
+// ----------------------------
+// Windows
+// ----------------------------
+%window {
+    @include clearfix();
+    margin-bottom: $baseline;
+    border: 1px solid $gray-lighter;
+    border-radius: ($baseline / 10);
+    box-shadow: 0 1px 1px $shadow;
+    background: $white;
+
+  // STATE: hover/active
+    &:hover, &:active {
+        box-shadow: 0 1px 1px $shadow;
+    }
+}
+
+
+// ----------------------------
+// Visual Disabled
+// ----------------------------
+%disabled {
+    pointer-events: none;
+    outline: none;
+    cursor: default;
+}
+
+// ----------------------------
+// Layering/Zindexing
+// ----------------------------
+%depth-0 { z-index: 1; }
+%depth-1 { z-index: 10; }
+%depth-2 { z-index: 100; }
+%depth-3 { z-index: 1000; }
+%depth-4 { z-index: 10000; }
+%depth-5 { z-index: 100000; }
+
+
+// ----------------------------
+// Clear Children
+// ----------------------------
+%clear-first-child {
+
+    &:first-child {
+        margin-top: 0;
+        padding-top: 0;
+        border-top: none;
+    }
+}
+
+%clear-last-child {
+
+    &:last-child {
+        margin-bottom: 0;
+        padding-bottom: 0;
+        border-bottom: none;
+    }
+}
+
+
+// ----------------------------
+// Lists
+// ----------------------------
+%list-unstyled {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-indent: 0;
+
+    li,
+    dt,
+    dd {
+        margin: 0;
+        padding: 0;
+    }
+}
+
+%list-inline {
+    @extend %list-unstyled;
+
+    li,
+    dt,
+    dd {
+        display: inline-block;
+    }
+}
+
+
+// ----------------------------
+// Icon Fonts
+// ----------------------------
+%icon-font {
+    display: inline-block;
+    min-width: 1rem; // Should be 16px
+    min-height: 1rem; // Should be 16px
+}
+
+
+// ----------------------------
+// Accessibility
+// ----------------------------
+%a11y-hide-text-image {
+    overflow: hidden;
+    text-indent: 100%;
+    white-space: nowrap;
+}
+
+%a11y-hide-text {
+    position: absolute;
+    overflow: hidden;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    padding: 0;
+    border: 0;
+    clip: rect(0 0 0 0);
+}
+
+
+// ----------------------------
+// Text Handling
+// ----------------------------
+%truncate-text {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+}
+
+abbr {
+    speak: spell-out;
```

Comment:
I did add it here though. I figured if there are ever any `abbr` (abbreviations), we can (theoretically) have screen readers spell-out the letters, rather than try to pronounce the word. Again, it's unreliable, but it can't hurt.